.github/workflows/django.yml

```diff
@@ -0,0 +1,29 @@
+name: django-testsuite
+on: [push, pull_request]
+jobs:
+  django-testsuite-3_1_7:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: django
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: ddtrace
+      - uses: actions/checkout@v2
+        with:
+          repository: django/django
+          ref: 3.1.7
```

Comment:
I wonder how we keep this up to date in the future so we always use the latest release.... do we want the latest release always?

I don't think any actions to take here, just something we haven't discussed yet, curious how we approach it.

Reply:
I think this is definitely a conversation worth having in standup, especially given #2190 and other framework tests in general 👍

Reply:
We need a tool to maintain / generate those files that is triggered by browsing releases on PyPI or something.

Reply:
I've just found this: https://github.com/marketplace/actions/find-latest-tag.

Reply:
> I've just found this: https://github.com/marketplace/actions/find-latest-tag.

☝🏻 I was going to suggest something similar. We can query `PyPI` in a step and use the result in next step